plugin/acl/acl.go

```diff
@@ -28,53 +28,60 @@ type Rule struct {
 	Policies []Policy
 }
 
+// Action defines the action against queries.
+type Action int
```

Comment:
Normally identifiers should not be capitalized unless you want they to be used by external packages

Reply:
Ah, I just capitalized it simply because I wanna use `action` as variable name. But will change this if it's of idiomatic Go style.

Reply:
Sorry but I'm a little bit confused. If I change `Rule`, `Action` and `Policy` to lowercase, naming variables seems to be inconvenient...

I cannot use `var rule Rule` any longer.

Reply:
Anyway. PR updated by changing all types to private.